recipe/build-lib.sh

```diff
@@ -12,7 +12,12 @@ if [ ${cuda_compiler_version} != "None" ]; then
     # docs.nvidia.com/cuda/cuda-compiler-driver-nvcc/index.html#gpu-feature-list
 
     ARCHES=(53 62 72)
-    if [ $(version2int $cuda_compiler_version) -ge $(version2int "11.1") ]; then
+    if [ $(version2int $cuda_compiler_version) -ge $(version2int "11.8") ]; then
+        # Hopper support for H100 (sm_90) needs cuda >= 11.8
+        LATEST_ARCH=90
+        # ARCHES does not contain LATEST_ARCH; see usage below
+        ARCHES=( "${ARCHES[@]}" 75 80 86)
```

Comment:
Depending on how we want to handle `sm_89` (if at all)

We could potentially add `sm_89` here

```suggestion
        ARCHES=( "${ARCHES[@]}" 75 80 86 89)
```

Alternatively we could replace `sm_86` with `sm_89`

```suggestion
        ARCHES=( "${ARCHES[@]}" 75 80 89)
```

Reply:
Given we are already running into CI time limits on Azure, think the answer needs to be picking `86` or `89`, but not both (at least not without dropping others)